app/views/registration/accountType.scala.html

```diff
@@ -43,7 +49,8 @@
                         content = Text(messages("accountType.orgOrAgent.helpText"))
                     ))
                 )
-            )
+            ),
+            errorMessage = fieldErrorMessage
```

Comment:
please try `.withFormField(form("accountTypeIndividual"))` ... there is an example here: https://github.com/hmrc/voa-property-linking-frontend/blob/248ff63e2442527527e5b505da021f98590d681a/app/views/propertyrepresentation/appoint/agentToManageMultipleProperties.scala.html#L50